src/command_modules/azure-cli-container/azure/cli/command_modules/container/_client_factory.py

```diff
@@ -0,0 +1,10 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+def _container_instance_client_factory(_):
+    from azure.mgmt.container import MicrosoftContainerInstanceManagementClient
```

Comment:
@yolo3301 Since fixed in the PR to be `ContainerInstanceManagementClient` please fix here as well. Thanks!